### DISCUSSION TITLE
State Management Handling

### DISCUSSION THREAD

**Discussion by wafiqmuhaz** [nylo-core/nylo#45] [Q&A] [answered]
Is there any handling of state management in Nylo similar to the bloc state management that has (bloc, event, and state)?

**Comment by agordn52** [comment, marked as answer]
Hi @wafiqmuhaz, 

You can manage the state using [NyState](https://nylo.dev/docs/4.x/ny-state) or any library of your choice.

Thanks,
Anthony
